### PR TITLE
Implement ban webhook

### PR DESCRIPTION
**Pull request by Salanto**
and you can actually turn it off!

### CODE REVIEW

File: core/include/discord.h [hunk 1]
```diff
@@ -91,6 +103,16 @@ public slots:
      */
     void onModcallWebhookRequested(const QString& f_name, const QString& f_area, const QString& f_reason, const QQueue<QString>& f_buffer);
 
+    /**
+     * @brief Handles a modcall webhook request.
```
Comment (in1tiate) [inline core/include/discord.h:107]: 🤔

File: core/src/discord.cpp [hunk 2]
```diff
@@ -56,6 +62,21 @@ QJsonDocument Discord::constructModcallJson(const QString &f_name, const QString
     return QJsonDocument(l_json);
 }
 
+QJsonDocument Discord::constructBanJson(const QString &f_ipid, const QString &f_moderator, const QString &f_duration, const QString &f_reason, const int &f_banID)
+{
+    QJsonObject l_json;
+    QJsonArray l_array;
+    QJsonObject l_object {
+        {"color", "13312842"},
+        {"title", "User has been banned by " + f_moderator},
```
Comment (in1tiate) [inline core/src/discord.cpp:71]:
```suggestion
        {"title", "Ban issued by " + f_moderator},
```
"User" is nonspecific and awkward

**Review by in1tiate — CHANGES_REQUESTED** [review]

File: core/include/discord.h (hunk 1 above)
Comment (Salanto) [reply core/include/discord.h]: image

**Review by in1tiate — APPROVED** [review]
looks fine

**Comment by in1tiate** [comment]
@scatterflower new feature, want your approval first

**Comment by scatterflower** [comment]
needs a squash

**Pull request merged**